oidc-service-impl/src/main/java/io/mosip/esignet/services/LinkedAuthorizationServiceImpl.java

```diff
@@ -217,12 +219,19 @@ public LinkedKycAuthResponseV2 authenticateUserV2(LinkedKycAuthRequest linkedKyc
         transaction.setKycToken(kycAuthResult.getKycToken());
         transaction.setAuthTimeInSeconds(IdentityProviderUtil.getEpochSeconds());
         authorizationHelperService.setIndividualId(linkedKycAuthRequest.getIndividualId(), transaction);
+        consentHelperService.processConsent(transaction, true);
         transaction.setProvidedAuthFactors(providedAuthFactors.stream().map(acrFactors -> acrFactors.stream()
                 .map(AuthenticationFactor::getType)
                 .collect(Collectors.toList())).collect(Collectors.toSet()));
-        cacheUtilService.setLinkedAuthenticatedTransaction(linkedKycAuthRequest.getLinkedTransactionId(), transaction);
+        if(ConsentAction.NOCAPTURE.equals(transaction.getConsentAction())){
+            cacheUtilService.setLinkedConsentedTransaction(transaction.getLinkedTransactionId(), transaction);
```

Comment:
validation of accepted claims  and permitted scopes is missing when directly transaction is directly added to linkedConsentedTransaction cache.

Reply:
OK, as the accepted claims and permitted scopes are added to the transaction in processConsent method itself, Marked this as resolved.